src/core/chatRoster.js

```diff
@@ -13,14 +13,21 @@
 
 /* global Candy */
 
+var $ = window.jQuery;
```

Comment:
Are you sure this is needed? You could just say `global Candy $` (or so) in line 14.